src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardSamlBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -48,19 +49,28 @@ class SamlProvider implements SamlProviderInterface, UserProviderInterface
      */
     private readonly array $administratorTeams;
 
+    private readonly string $surfConextResponsibleAuthorization;
+
     public function __construct(
         private readonly ContactRepository $contacts,
         private readonly ServiceRepository $services,
         private readonly AttributeDictionary $attributeDictionary,
-        private readonly LoggerInterface $logger,
+        private readonly LoggerInterface  $logger,
         string $administratorTeams,
+        string $surfConextResponsibleAuthorization,
```

Comment:
Why not using property promotion here?

Reply:
I feel it more logical to first perform the input validation and then set the value. It can be done in retrospect i guess. But this felt most natural to me.